docs/csharp/getting-started/index.md

```diff
@@ -32,3 +51,33 @@ The following topics are available:
    Visual Studio Code is a free code editor optimized for building and debugging modern web and cloud applications. It supports IntelliSense and is available for Linux, macOS, and Windows.
 
    This topic shows you how to create and run a simple Hello World application with Visual Studio Code and .NET Core.
+
+* [Additional Resources for Visual C# Programmers](additional-resources.md)
+
+   Provides links to Web sites and newsgroups that can help you find answers to common problems.
+
+## Related Sections
+
+* [Using the Visual Studio Development Environment for C#](/visualstudio/csharp-ide/using-the-visual-studio-development-environment-for-csharp)  
+
+    Provides a guide to using the [!INCLUDE[csprcs](~/includes/csprcs-md.md)] integrated development environment.
+
+* [C# Programming Guide](../../csharp/programming-guide/index.md)
+
+    Provides information about C# programming concepts, and describes how to perform various tasks in C#.
+
+* [C# Reference](../../csharp/language-reference/index.md)
+
+    Provides detailed reference information about C# keywords, operators, preprocessor directives, compiler options, and compiler errors and warnings.
+
+* [Visual Studio Samples](/visualstudio/ide/visual-studio-samples)
+
+    Provides information about how you can access online samples.
+
+* [Walkthroughs](../../csharp/walkthroughs.md)
+
+    Provides links to programming walkthroughs that use C# and a brief description of each walkthrough.
+
+## See Also
+ [Getting Started with Visual C# and Visual Basic](/visualstudio/ide/getting-started-with-visual-csharp-and-visual-basic)
```

Comment:
this link caught me by surprise. we should probably do a PR on the VS repo to add VS to the title.

Reply:
Good idea.  See https://github.com/MicrosoftDocs/visualstudio-docs/pull/266